feat(documentation): January 2023 newsletter. (#6000)
jenkins-infra/jenkins.io: content/blog/2023/02/2023-02-05-jenkins-newsletter.adoc

==  Highlights

* Jenkins in GSoC planning is in full steam ahead.
* General availability of new development tools on ci.jenkins.io: Maven, JDK, Playwright.
* 98 pull requests were merged from 38 different authors in January.
* Jenkins 2.375.2 released January 11, 2023.
Over 350 positive ratings.
* A sandbox bypass vulnerability was corrected among 37 other vulnerabilities.
The security team recommends users to upgrade.
* Debian 12 (“bookworm”) will not deliver OpenJDK 11

[[outreach]]
image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]

Contributed by: author:alyssat[Alyssa Tong]

* The GSoC Org admin team kicked the new year off with all things GSoC!

** We hosted a https://youtu.be/gGTZtKjVlK0[webinar] for new mentors.
** Finalized the https://www.jenkins.io/projects/gsoc/2023/project-ideas/[GSoC 2023
project ideas list.]
** https://www.jenkins.io/blog/2023/02/01/gsoc-update/[This post] encapsulates current progress and what’s next.
** There’s still time to
https://community.jenkins.io/t/jenkins-in-gsoc-2023-mentors-org-admins-project-ideas-wanted/4387[become a mentor and or propose project idea(s)].

> * FOSDEM https://docs.google.com/document/d/1ajANRpnhn0akwZV0L1S6qtqDMgK5TRUs5_8wUZYzcZg/edit?usp=sharing[planning] in full force!

Reviewer: ```suggestion
* FOSDEM https://docs.google.com/document/d/1ajANRpnhn0akwZV0L1S6qtqDMgK5TRUs5_8wUZYzcZg/edit?usp=sharing[planning] in full force!
```

Reviewer: Let's remove this bullet item

> It’s exciting to be back for this in-person event this  year.

Reviewer: ```suggestion
It’s exciting to be back for this in-person event this  year.
```

Reviewer: And remove this bullet item too, plz

---
layout: post
title: "Jenkins January 2023 Newsletter"
tags:
- jenkins
- newsletter
- community
- contribute
authors:
- alyssat
- kmartens27
opengraph:
image: /images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png
links:
discourse: true
---

image:/images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png[Jenkins January Newsletter]

==  Highlights

* Jenkins in GSoC planning is in full steam ahead.
* General availability of new development tools on ci.jenkins.io: Maven, JDK, Playwright.
* 98 pull requests were merged from 38 different authors in January.

> * Jenkins 2.375.2 released January 11, 2023.  Over 350 positive ratings.
> * A sandbox bypass vulnerability was corrected among 37 other vulnerabilities. The security team recommends users to upgrade.

Reviewer 2: ```suggestion
* Jenkins 2.375.2 released January 11, 2023.
Over 350 positive ratings.
* A sandbox bypass vulnerability was corrected among 37 other vulnerabilities.
The security team recommends users to upgrade.
```
single sentence per line formatting suggestion

* Debian 12 (“bookworm”) will not deliver OpenJDK 11

[[outreach]]
image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]

Contributed by: author:alyssat[Alyssa Tong]

* The GSoC Org admin team kicked the new year off with all things GSoC!

** We hosted a https://youtu.be/gGTZtKjVlK0[webinar] for new mentors.
** Finalized the https://www.jenkins.io/projects/gsoc/2023/project-ideas/[GSoC 2023
project ideas list.]
** https://www.jenkins.io/blog/2023/02/01/gsoc-update/[This post] encapsulates current progress and what’s next.
** There’s still time to
https://community.jenkins.io/t/jenkins-in-gsoc-2023-mentors-org-admins-project-ideas-wanted/4387[become a mentor and or propose project idea(s)].

> * FOSDEM
> https://docs.google.com/document/d/1ajANRpnhn0akwZV0L1S6qtqDMgK5TRUs5_8wUZYzcZg/edit?usp=sharing[planning]
> in full force! It’s exciting to be back for this in-person event this  year.

Reviewer 2: ```suggestion
* FOSDEM https://docs.google.com/document/d/1ajANRpnhn0akwZV0L1S6qtqDMgK5TRUs5_8wUZYzcZg/edit?usp=sharing[planning] in full force!
It’s exciting to be back for this in-person event this  year.
```
single sentence per line formatting

image:/images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png[Infrastructure Update]

Contributed by: author:dduportal[Damien Duportal]

* Bumped our controllers to LTS 2.375.2
* General availability of new development tools on ci.jenkins.io:
** Maven 3.8.7
** JDK 8u362-b09
** JDK 11.0.18+10
** JDK 17.0.6+10
** JDK 19.0.2+7
** Playwright (headless web-browser testing)

> * Started to decrease download bandwidth from JFrog (repo.jenkins-ci.org)  to our infrastructure by enabling artifact caching for plugins builds on  ci.jenkins.io
> ** A lot of “under-the-hood” network and performances related work
> * Cleaned up deprecated plugins from all of our controllers (momentjs,  jquery bundle and ace-editor bundle)

Reviewer 2: ```suggestion
* Started to decrease download bandwidth from JFrog (repo.jenkins-ci.org)  to our infrastructure by enabling artifact caching for plugins builds on  ci.jenkins.io.
** A lot of “under-the-hood” network and performances related work.
* Cleaned up deprecated plugins from all of our controllers (momentjs, jquery bundle and ace-editor bundle).
```
adding full stops

[[documentation]]
image:/images/post-images/2023/01/12/jenkins-newsletter/jenkins-io-improvements.png[Jenkins io improvements Update]

Contributed by: author:kmartens27[Kevin Martens]

During the month of January, 98 pull requests were merged from 38  different authors.
We are also preparing for the 2023 edition of Google  Summer of Code, by encouraging folks to share ideas or join as mentors.

> We published one blog post, which was the 2022 recap and contained tons
> of highlights for the last year of the Jenkins project.

Reviewer 2: ```suggestion
We published one blog post, which was the link:/blog/2023/01/12/jenkins-newsletter/[2022 recap] and contained tons of highlights for the last year of the Jenkins project.
```
adding link to post and single sentence per line formatting

Thanks to all of the new and returning contributors for their hard work already in the new year, and the continued efforts that are bound to happen.

[[Governance]]
image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]

Contributed by: author:markewaite[Mark Waite]

Jenkins 2.375.2 released January 11, 2023.
Over 350 positive https://www.jenkins.io/changelog-stable/#v2.375.2[ratings] with only 1  issue reported to have required a rollback for 4 users.

> The most recent 9 Jenkins weekly releases (2.381 - 2.389) have received
> positive https://www.jenkins.io/changelog/#v2.389[ratings] as well,
> with 40-50 positive ratings and only 1 reported rollback in each of
> those 9 weekly releases.

Reviewer 2: ```suggestion
The nine most recent Jenkins weekly releases (2.381 - 2.389) have received positive link:/changelog/#v2.389[ratings] as well, with 40-50 positive ratings and only 1 reported rollback in each of those 9 weekly releases.
```
updated line formatting and updating link

> Thanks to everyone that has done such great work on Jenkins core and its
> recent releases.

Reviewer 2: ```suggestion
Thanks to everyone that has done such great work on Jenkins core and its recent releases.
```
single sentence per line formatting suggestion

[[platform]]
image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]

Contributed by: author:gounthar[Bruno Verachten]

* Docker images
** New platforms:

> *** UBI9 with JDK17 proposed and maintained by Oliver Gondza from RedHat
> (also maintainer of UBI8 container image)
> *** https://github.com/jenkins-infra/helpdesk/issues/3318[Zombie
> images] fixed. Hurray!
> *** Java updates were deployed to infra and container images (except for
> Windows containers)
> **** 8u362
> **** 11.0.18
> **** 17.0.6
> **** 19.0.2
> * Debian 12 (“bookworm”) will not deliver OpenJDK 11
> ** End of life date for Debian’s openJDK11 won’t happen until 2026 or
> 2027
> ** No urge to drop jdk11 support for Jenkins
> ** Will change the documentation nonetheless when it goes out, so that  we describe the use of Jenkins with openJDK17
> * JDK Support for Jenkins
> ** Jenkins' enhancement proposal for required JDK11 is final! Congrats.

Reviewer 2: ```suggestion
*** UBI9 with JDK17 proposed and maintained by Oliver Gondza from RedHat, who's also maintainer of the UBI8 container image.
*** https://github.com/jenkins-infra/helpdesk/issues/3318[Zombie images] fixed. Hurray!
*** Java updates were deployed to infra and container images (except for Windows containers):
**** 8u362
**** 11.0.18
**** 17.0.6
**** 19.0.2
* Debian 12 (“bookworm”) will not deliver OpenJDK 11.
** The end of life date for Debian’s openJDK11 won’t happen until 2026 or 2027.
** There is no urge to drop jdk11 support for Jenkins.
** We will change the documentation nonetheless when it goes out, so that  we describe the use of Jenkins with openJDK17.
* JDK Support for Jenkins
** Jenkins' enhancement proposal for required JDK11 is final!
Congrats.
```
single sentence per line and full stop suggestions

[[modern-ui]]
image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]

Contributed by: author:markewaite[Mark Waite]

The Jenkins user experience continues to improve. Recent weekly releases have included hiding of potentially sensitive values in system properties and in environment variables, navigation improvements with more breadcrumb entries, changes to the plugin manager, and internal updates to various libraries.

[[security-fixes]]
image:/images/post-images/2023/01/12/jenkins-newsletter/security.png[Security Update]

Contributed by: author:wadeck[Wadeck Follonier]

> A sandbox bypass vulnerability was corrected and announced in the https://www.jenkins.io/security/advisory/2023-01-24/[January security advisory] among 37 others vulnerabilities. The security team recommends users to upgrade.

Reviewer 2: ```suggestion
A sandbox bypass vulnerability was corrected and announced in the https://www.jenkins.io/security/advisory/2023-01-24/[January security advisory] among 37 others vulnerabilities.
The security team recommends users to upgrade.
```
single sentence per line formatting
